browser/src/canvas/sections/ShapeHandlesSection.ts

```diff
@@ -911,7 +911,16 @@ class ShapeHandlesSection extends CanvasSectionObject {
 	}
 
 	onMouseMove(position: number[], dragDistance: number[]) {
-		if (this.containerObject.isDraggingSomething() && !app.file.textCursor.visible) {
+		let canDrag = !app.file.textCursor.visible;
+
+		if (canDrag && app.map._docLayer._docType === 'presentation') {
+			// Tables get selected when multiple cells are selected. In this case, we check if DeleteRows is disabled.
+			// Because in non-edit mode, deleteRows is disabled. So we can drag the table.
+			const deleteRowsState = app.map['stateChangeHandler']._items['.uno:DeleteRows'];
+			canDrag = deleteRowsState ? deleteRowsState === 'disabled': true;
```

Comment:
this is received with a delay... smells like a race here
don't we get any better infromation together with message wrt these shape handles? graphic selection possibly is the command?

Reply:
Did you test it? When user starts editing, deleteRows is disabled and it doesn't get enabled unless user stops editing and selects the table. I checked it now and it seems to work well.

DeleteRows is an indicator. Graphic selection command is not sent again while a table is already selected. I could pick something else but this also works and also code is commented.

Reply:
no, I just read the code and noticed. I belive it works. But in general there is no warranty at which point in the time uno command updates will be received I think.

So best to avoid relying on that if possible. But fine for now.